Resources/de/Resources.resw

```diff
@@ -118,6 +118,671 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="NavigationBar_Mixes" xml:space="preserve">
-    <value>Mixes</value>
+    <value>Mischungen</value>
+  </data>
+  <data name="OnboardingWizard_SetupDevice_Title" xml:space="preserve">
+    <value>Richten Sie Ihr {0} ein</value>
+  </data>
+  <data name="OnboardingWizard_SetupDevice_Description" xml:space="preserve">
+    <value>Lassen Sie uns eine kurze Einrichtung durchgehen, um sicherzustellen, dass alles so funktioniert, wie Sie es möchten – und genau richtig klingt.</value>
+  </data>
+  <data name="OnboardingWizard_GetStarted" xml:space="preserve">
+    <value>Erste Schritte</value>
+  </data>
+  <data name="OnboardingWizard_Skip" xml:space="preserve">
+    <value>Überspringen</value>
+  </data>
+  <data name="OnboardingWizard_ExploreDevice_Title" xml:space="preserve">
+    <value>Erkunden Sie Ihr Gerät</value>
+  </data>
+  <data name="OnboardingWizard_MuteMic_Title" xml:space="preserve">
+    <value>Stummschalten Sie Ihr Mikrofon sofort</value>
```

Comment:
This translations sounds weird in German. Shouldn't it be rather something like "Schalten Sie Ihr Mikrofon sofot stumm"?